src/sensors/odometer/interrupt/DirectionlessOdometer.cpp

```diff
@@ -1,4 +1,4 @@
-#include <math.h> // NOLINT(modernize-deprecated-headers)
+#include <math.h>
```

Comment:
:warning: **modernize-deprecated-headers** :warning:
inclusion of deprecated C++ header 'math.h'; consider using 'cmath' instead
```suggestion
#include <cmath>```